src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -1828,6 +1838,52 @@ protected function has_fatal_error() {
 			$result       = json_decode( trim( $error_output ), true );
 		}
 
+		if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+			$fatal_error = null;
+
+			$last_error = error_get_last();
+			if ( $last_error && in_array( $last_error['type'], array( E_ERROR, E_PARSE, E_COMPILE_ERROR, E_USER_ERROR ) ) ) {
+				$fatal_error = "PHP Fatal error: {$last_error['message']} in {$last_error['file']} on line {$last_error['line']}";
+			} else {
+				$log_file = WP_CONTENT_DIR . '/debug.log';
```

Comment:
This should use whatever the `error_log` is set to in the PHP config.

```suggestion
				$log_file = ini_get( 'error_log' );
```

For another approach at parsing the error log, consider this code from the AMP plugin: https://github.com/ampproject/amp-wp/blob/2.5.4/src/Support/SupportData.php#L351-L410